controllers/imageclusterinstall_controller_test.go

```diff
@@ -107,11 +122,10 @@ var _ = Describe("Reconcile", func() {
 			Domain:          "example.com",
 			ClusterName:     "thingcluster",
 			MasterIP:        "192.0.2.1",
-			ReleaseRegistry: "registry.example.com",
+			ReleaseRegistry: "registry.example.com/releases",
```

Comment:
LCA get the reigstry [this way](https://github.com/openshift-kni/lifecycle-agent/blob/9ff71276db124e563af5adad60f35f53eb15241a/utils/client_helper.go#L190), meaning in LCA will not expect ot the `/release` in the registry name

Reply:
Okay so it only wants the registry hostname?

Reply:
That would make this a lot less error prone actually

Reply:
Done